docs/quickstart-mission.md

```
@@ -0,0 +1,348 @@
+```
+
+## Step 6: Generate a Design Plan
+
+Create a static design plan that lays out the compute pipeline.
+
+```bash
+branes design plan --mission vineyard-sprayer --static
+```
+
+Expected output:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)